src/item.cpp

```diff
@@ -209,6 +209,29 @@ item::item( const itype *type, time_point turn, solitary_tag )
 item::item( const itype_id &id, time_point turn, solitary_tag tag )
     : item( find_type( id ), turn, tag ) {}
 
+item::item( const recipe *rec, long qty, t_item_vector items )
+    : item( "craft", calendar::turn, qty )
+{
+    making = rec;
+    components = items;
```

Comment:
Assignments like this are better done in the initialization list.

Reply:
Actually can't in this case, this is a "Delegating constructor."  See https://en.cppreference.com/w/cpp/language/initializer_list

Reply:
Yeah, you're right. It's the same class, so can't indeed.